Teleportation/ReferenceImplementation.qs

```diff
@@ -74,6 +74,14 @@ namespace Quantum.Kata.Teleportation {
         return Measure([basis], [qBob]) == One;
     }
     
+    // Task 1.8. Entanglement swapping
+    operation EntanglementSwapping_Reference (entangledPairAlice : Qubit[], entangledPairBob : Qubit[]) : Unit {
+        CNOT(entangledPairAlice[0], entangledPairBob[0]);
```

Comment:
Can we use SendMessage_Reference here instead of the individual gates to emphasize that it's the same protocol?

Reply:
yes - changed that, thanks!